src/char/int_homun.cpp

```diff
@@ -142,6 +141,25 @@ bool mapif_homunculus_save(struct s_homunculus* hd)
 		}
 	}
 
+	// Save skill cooldowns
+	SqlStmt *stmt = SqlStmt_Malloc(sql_handle);
+
+	if (SQL_ERROR == SqlStmt_Prepare(stmt, "REPLACE INTO `%s` (`homun_id`, `skill`, `tick`) VALUES (%d, ?, ?)", schema_config.skillcooldown_homunculus_db, hd->hom_id))
+		SqlStmt_ShowDebug(stmt);
```

Comment:
If it fails to prepare, it does not really make sense to continue?
But I see that this is already wrong in the code where you copied it from.